sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/UnknownIdentifier.java

```diff
@@ -17,12 +17,12 @@ public final class UnknownIdentifier extends CommunicationIdentifier {
      * @param id the string identifier representing the identity
      * @throws IllegalArgumentException thrown if id parameter fail the validation.
      */
-    public UnknownIdentifier(String id) {
+    protected UnknownIdentifier(String id) {
```

Comment:
```suggestion
    public UnknownIdentifier(String id) {
```

Reply:
this needs to stay public

Reply:
@petrsvihlik This method is a new addition, so it can be protected.
<img width="453" alt="image" src="https://user-images.githubusercontent.com/1885005/182677280-15a75305-8533-4946-9700-49953146f09f.png">